docs/astro.config.mjs

```diff
@@ -57,6 +57,10 @@ const locales = {
 		label: 'العربية',
 		lang: 'ar',
 	},
+	kg: {
```

Comment:
It would be nice to include some translations as part of this change even if it's just for the index.mdx. Otherwise this change won't provide as much value to users since it'll just add the item to the translations drop-down without having any translated content.

Reply:
I will add translations for the pages in the future.